pkg/transform/mapper.go

```diff
@@ -112,7 +112,14 @@ func jqMapperConfigFunction(c *JQMapperConfig) (TransformationFunction, error) {
 		if a1 == nil {
 			return nil
 		}
-		return a1.(time.Time).UnixMilli()
+
+		validTime, ok := a1.(time.Time)
+
+		if !ok {
+			return fmt.Errorf("Not a valid time input to 'epoch' function")
```

Comment:
I think we should use errors.New [like we do elsewhere](https://github.com/snowplow/snowbridge/blob/c8c94f8a13367260f9a1a280cdd5d86098f66521/pkg/transform/snowplow_enriched_set_pk.go#L49).

Reply:
Also shouldn't it be nil, error?

Reply:
I think no, this function only has one return value with type `any`.

Reply:
You're right - in the diff I missed that it's in the jq function definition, not the transformation itself. Was wondering why it compiled! :D